### PR TITLE
Sscs 9588 kustomize hearings api

### PR DESCRIPTION
**Pull request by dattipoeHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-9588

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/sscs/sscs-bulk-scan/aat.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: sscs-bulk-scan
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/sscs/bulk-scan:prod-0b474b1-20220328103050 #{"$imagepolicy": "flux-system:sscs-bulk-scan"}
```
Comment (chrisorisawayi) [inline k8s/namespaces/sscs/sscs-bulk-scan/aat.yaml:8]: Image should be in base Helm Release if not overriding prod image.

Comment (chrisorisawayi) [inline k8s/namespaces/sscs/sscs-bulk-scan/aat.yaml:8]: Image should be in base Helm Release if not overriding prod image.

File: k8s/namespaces/sscs/sscs-bulk-scan/perftest.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: sscs-bulk-scan
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/sscs/bulk-scan:prod-0b474b1-20220328103050 #{"$imagepolicy": "flux-system:sscs-bulk-scan"}
```
Comment (chrisorisawayi) [inline k8s/namespaces/sscs/sscs-bulk-scan/perftest.yaml:8]: Image should be in base Helm Release if not overriding prod image.

Comment (chrisorisawayi) [inline k8s/namespaces/sscs/sscs-bulk-scan/perftest.yaml:8]: Image should be in base Helm Release if not overriding prod image.

File: k8s/namespaces/sscs/sscs-hearings-api/ithc.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: sscs-hearings-api
+  namespace: sscs
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/sscs/hearings-api:prod-2d34469-20220325150900 #{"$imagepolicy": "flux-system:sscs-hearings-api"}
```
Comment (chrisorisawayi) [inline k8s/namespaces/sscs/sscs-hearings-api/ithc.yaml:9]: Image should be in base Helm Release if not overriding prod image.  Make similar changes in applicable environments.

**Review by thomast1906 — CHANGES_REQUESTED** [review]
AS prod is not to be included, can you remove prod references.

also missing base sscs as per discussion. Example to show what is required: https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/camunda/kustomization.yaml#L5-L7

**Review by thomast1906 — CHANGES_REQUESTED** [review]
AS prod is not to be included, can you remove prod references.

also missing base sscs as per discussion. Example to show what is required: [https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/camunda/kustomization.yaml#L5-L7](https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/camunda/kustomization.yaml?rgh-link-date=2022-04-05T12%3A31%3A30Z#L5-L7)

File: k8s/aat/common-overlay/sscs/kustomization.yaml [hunk 4]
```diff
@@ -4,6 +4,9 @@ namespace: sscs
 bases:
 - ../../../namespaces/sscs
 - ../../../namespaces/all-namespaces/nonprod-role.yaml
+- ../../../namespaces/sscs/sscs-hearings-api/sscs-hearings-api.yaml
```
Comment (thomast1906) [inline k8s/aat/common-overlay/sscs/kustomization.yaml:7]: I am assuming once all apps are kustomized - you will be moving this into namespaces/sscs - rather thn a specific per each app

**Review by thomast1906 — APPROVED** [review]

**Review by ss-evoco — APPROVED** [review]

**Pull request merged**